### DISCUSSION TITLE
Scrolling disabled when using bootstrap + drag and drop

### DISCUSSION THREAD

**Discussion by PaddyWitzigmann** [twbs/bootstrap#35785] [Q&A] [answered]
I'm having a problem that I can't get rid of when using bootstrap together with drag and drop (library: dnd-kit).

When I'm adding bootstrap into the project the window isn't scrolling anymore as soon as I start dragging an element up or down. Here is a sandbox illustrating the problem: https://codesandbox.io/s/dnd-kit-bootstrap-scrolling-bug-u3jg2?file=/src/index.js Comment out the bootstrap import in the index.js file in line 3 to see the difference.

Do you have any idea how I can solve this? Thanks a lot for your help!

**Comment by PaddyWitzigmann** [comment, marked as answer]
It can be fixed like this: 

@import "../node_modules/bootstrap/scss/bootstrap";

//SOLVING SCROLLING BUG: when "scroll-behavior: smooth;" is set like in the original bootstrap template scrolling isn't working correctly together with DnD
//Hence this needs to be overwritten with "scroll-behavior: auto;"
:root {
  @if $enable-smooth-scroll {
    @media (prefers-reduced-motion: no-preference) {
      scroll-behavior: auto;
    }
  }
}

**Reply by dt-karl-kuhn** [reply]
Hi, I have a single page application using React and bootstrap with dnd-kit. How might I apply this in that situation.
